kvariants/build.rs

```diff
@@ -0,0 +1,38 @@
+use serde::Deserialize;
+use std::error::Error;
+
+#[derive(Deserialize)]
+pub struct TsvRow {
+    lhs: String,
+    relation: String,
+    rhs: String,
+}
+
+fn main() {
+    // Tell Cargo that if the given file changes, to rerun this build script.
+    println!("cargo:rerun-if-changed=dictionaries/source/kVariants.tsv");
```

Comment:
It may be interesting to add these 2 additional checks.
```suggestion
    println!("cargo:rerun-if-changed=dictionaries/source/kVariants.tsv");
    println!("cargo:rerun-if-changed=build.rs");
    println!("cargo:rerun-if-changed=Cargo.toml");
```

Reply:
Fixed in cd6c975